testsuite/MDAnalysisTests/lib/test_distances.py

```diff
@@ -1022,23 +1052,58 @@ def test_dihedrals_wronglength(self, positions, wronglength, backend):
         with pytest.raises(ValueError):
             distances.calc_dihedrals(a, b, c, wronglength, backend=backend)
 
+    @pytest.mark.parametrize("backend", ["serial", "openmp"])
     def test_dihedrals_bad_result(self, positions, backend):
         a, b, c, d = positions
         badresult = np.zeros(len(a) - 1)  # Bad result array
 
         with pytest.raises(ValueError):
             distances.calc_dihedrals(a, b, c, d, result=badresult, backend=backend)
 
-    @pytest.mark.parametrize('case', [
-        (np.array([[1, 2, 1], [1, 1, 1], [2, 1, 1], [2, 2, 1]], dtype=np.float32), 0.),  # 0 degree angle (cis)
-        (np.array([[1, 2, 1], [1, 1, 1], [2, 1, 1], [2, 0, 1]], dtype=np.float32), np.pi),  # 180 degree (trans)
-        (np.array([[1, 2, 1], [1, 1, 1], [2, 1, 1], [2, 1, 2]], dtype=np.float32), 0.5 * np.pi),  # 90 degree
-        (np.array([[1, 2, 1], [1, 1, 1], [2, 1, 1], [2, 1, 0]], dtype=np.float32), 0.5 * np.pi),  # other 90 degree
-        (np.array([[1, 2, 1], [1, 1, 1], [2, 1, 1], [2, 2, 2]], dtype=np.float32), 0.25 * np.pi),  # 45 degree
-        (np.array([[1, 2, 1], [1, 1, 1], [2, 1, 1], [2, 0, 2]], dtype=np.float32), 0.75 * np.pi),  # 135
-    ])
-    @pytest.mark.parametrize('shift', shifts)
-    @pytest.mark.parametrize('periodic', [True, False])
+    @pytest.mark.parametrize(
+        "case",
+        [
+            (
+                np.array(
+                    [[1, 2, 1], [1, 1, 1], [2, 1, 1], [2, 2, 1]], dtype=np.float32
+                ),
+                0.0,
+            ),  # 0 degree angle (cis)
+            (
+                np.array(
+                    [[1, 2, 1], [1, 1, 1], [2, 1, 1], [2, 0, 1]], dtype=np.float32
+                ),
+                np.pi,
+            ),  # 180 degree (trans)
+            (
+                np.array(
+                    [[1, 2, 1], [1, 1, 1], [2, 1, 1], [2, 1, 2]], dtype=np.float32
+                ),
+                0.5 * np.pi,
+            ),  # 90 degree
+            (
+                np.array(
+                    [[1, 2, 1], [1, 1, 1], [2, 1, 1], [2, 1, 0]], dtype=np.float32
+                ),
+                0.5 * np.pi,
+            ),  # other 90 degree
+            (
+                np.array(
+                    [[1, 2, 1], [1, 1, 1], [2, 1, 1], [2, 2, 2]], dtype=np.float32
+                ),
+                0.25 * np.pi,
+            ),  # 45 degree
+            (
+                np.array(
+                    [[1, 2, 1], [1, 1, 1], [2, 1, 1], [2, 0, 2]], dtype=np.float32
+                ),
+                0.75 * np.pi,
+            ),  # 135
+        ],
+    )
```

Comment:
Is there actually any change here except from conforming to `black`? this 100% makes readability worse, I'd ask that we go back to the previous formatting even style if `black` complains.